addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -837,7 +837,7 @@ var WebsiteLivechatMessage = AbstractMessage.extend({
     getAvatarSource: function () {
         var source = this._serverURL;
         if (this.hasAuthor()) {
-            source += '/web/partner_image/' + this.getAuthorID();
+            source += '/web/partner_image/' + this.getAuthorID() + '/avatar_128';
```

Comment:
This route does not take a field name as param, and also it was used to have a specific placeholder which is now obsolete, so I guess the route can be deleted and replaced by the normal one?